repos/common.proj

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
   <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)src/Microsoft.Extensions.CommandLineUtils/bin/$(Configuration)/</PackagesOutput>
-    <RepoApiImplemented>false</RepoApiImplemented>
+    <RepositoryOrganization>aspnet</RepositoryOrganization>
+    <RepositoryBranch>servicing/1.0.x</RepositoryBranch>
+    <PackagesOutput>$(ProjectDirectory)artifacts/build/</PackagesOutput>
+    <BuildArguments>/p:Configuration=$(Configuration)</BuildArguments>
+    <BuildArguments Condition=" '$(OS)' != 'Windows_NT' ">$(BuildArguments) /p:SkipNETFramework=true</BuildArguments>
```

Comment:
You should consider setting SkipNETFramework to true in your repo based on DotNetBuildFromSource being set to true https://github.com/dotnet/source-build/blob/dev/release/2.0/repos/dir.props#L27.

Reply:
I originally did that, but wasn't sure if there might be upstream consumers also compiling .NET Framework assets in source-build. Is it safe to assume DotNetBuildFromSource=true means .NET Framework can always be skipped, even on windows?

Reply:
If you need it only on non-windows then you can still do your OS check in the common repo as well. It just moves the responsibility to the repo instead of source-build needing to know its custom logic. This is going to be the model as we get closer to just using the repo API.

Reply:
Ok, I updated to move this into aspnet/common instead.